sdks/python/apache_beam/coders/slow_stream.py

```diff
@@ -34,15 +34,18 @@ class OutputStream(object):
 
   def __init__(self):
     self.data = []
+    self.byte_count = 0
```

Comment:
Nit: Shall we rely on ` len(self.data)` instead of creating and managing a new field.

Reply:
Unfortunately we can't, as the elements of data are of varying size.